mitodl/bootcamp-ecommerce
# Bootcamp enrollments models

#### What are the relevant tickets?
Fixes #468 

#### What's this PR do?
Sets up `BootcampRunEnrollment` to track enrollments. There is also code to update the application state to `COMPLETE` when all payments are received.

#### How should this be manually tested?
Pay for a course, but enter a dollar amount below the cost of the course. You should not see a `BootcampRunEnrollment` created for that run. Pay the extra value, and you should see that it exists in the database for the run and user.
